vortexor/src/vortexor.rs

```diff
@@ -88,6 +92,21 @@ impl Vortexor {
         }
     }
 
+    pub fn create_sigverify_stage(
+        tpu_receiver: Receiver<solana_perf::packet::PacketBatch>,
+        non_vote_sender: TracedSender,
+    ) -> SigVerifyStage {
+        // Not interesed of banking tracing
```

Comment:
probably typo: of -> in

Reply:
Also not sure what purpose this comment serves.

Reply:
Fixed